### PR TITLE
Fixes configuration for redirect post-login

### PR DESCRIPTION
**Pull request by CrisTofani**

### CODE REVIEW

File: .env.uat [hunk 1]
```diff
@@ -10,7 +10,7 @@ MSAL_CLIENT_ID=2416d411-9cbb-4d4c-a902-2570f031b9f0
 
 MSAL_AUTHORITY=https://cgnonboardingportaluat.b2clogin.com/cgnonboardingportaluat.onmicrosoft.com/B2C_1_login/
 
-MSAL_REDIRECT_URI=https://cgnonboardingportal-u-cdnendpoint-frontend.azureedge.net/
+MSAL_REDIRECT_URI=https://https://portal.cgnonboardingportal-uat.pagopa.it/
```
Comment (CrisTofani) [inline .env.uat:13]:
```suggestion
MSAL_REDIRECT_URI=https://portal.cgnonboardingportal-uat.pagopa.it/
```

**Review by AleDore — APPROVED** [review]
LGTM but... only for UAT env?

**Comment by CrisTofani** [comment]
> [No Coverage information](https://sonarcloud.io/component_measures?id=pagopa_cgn-onboarding-portal-frontend&pullRequest=172) No Coverage information

this is just a check, both UAT and PROD env vars have raw URL for portal which differs from the real one (EG. https://cgnonboardingportal-u-cdnendpoint-frontend.azureedge.net/ instead of https://portal.cgnonboardingportal-uat.pagopa.it/  for UAT environment)

**Comment by CrisTofani** [comment]
> LGTM but... only for UAT env?

this is just a check, both UAT and PROD env vars have raw URL for portal which differs from the real one (EG. https://cgnonboardingportal-u-cdnendpoint-frontend.azureedge.net/ instead of https://portal.cgnonboardingportal-uat.pagopa.it/ for UAT environment)

**Pull request merged**